packages/neuron-ui/src/components/TransferConfirm/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Card, Button, InputGroup, FormControl } from 'react-bootstrap'
+import { useTranslation } from 'react-i18next'
+
+const TransferConfirm = ({
+  title,
+  message,
+  psw,
+  onChange,
+  onSubmit,
+  onCancel,
+}: {
+  title: string
+  message: string | React.ReactNode
+  psw: string
+  onChange: any
+  onSubmit: any
+  onCancel: any
+}) => {
+  const [t] = useTranslation()
+  return (
+    <Card
+      onClick={(e: React.SyntheticEvent<HTMLDivElement>) => {
+        e.preventDefault()
+        e.stopPropagation()
+      }}
+    >
+      <Card.Header>{title}</Card.Header>
+      <Card.Body>
+        <Card.Text>{`${t('send.thistransferwillsend')}: `}</Card.Text>
+        {typeof message === 'string' ? <Card.Text>{message}</Card.Text> : message}
+        <InputGroup>
+          <FormControl placeholder={t('send.inputpasswordtoconfirm') as string} value={psw} onChange={onChange} />
```

Comment:
I have to confess that lower cased key without separator like `-` or `_` is not readable. Which would be  better, `send.inputPasswordToConfirm` or `send.input-password-to-confirm`?

Reply:
In a previous discussion, we decide to use lowercase, so I prefer `send.input-password-to-confirm`

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/156/commits/e8c2d4d2f1c7d73947c74eb1b1e8f651595339ef